instamatic/processing/ImgConversion.py

```diff
@@ -647,6 +647,51 @@ def write_pets_inp(self, path: str, tiff_path: str = 'tiff') -> None:
                 angle = self.start_angle + sign * self.osc_angle * i
                 print(f'{tiff_path}/{fn} {angle:10.4f} 0.00', file=f)
             print('endimagelist', file=f)
+    
+    def write_pets2_inp(self, path: str, tiff_path: str = 'tiff') -> None:
+        """Write PETS 2 input file `pets.pts2` in directory `path`"""
+        path.mkdir(exist_ok=True, parents=True)
+
+        if self.start_angle > self.end_angle:
+            sign = -1
+        else:
+            sign = 1
+
+        omega = np.degrees(self.rotation_axis)
+
+        with open(path / 'pets.pts2', 'w') as f:
+            date = str(time.ctime())
+            print(f'# PETS 2 input file for Rotation Electron Diffraction generated by `instamatic`', file=f)
+            print(f'# {date}', file=f)
+            print(f'# For definitions of input parameters, see:', file=f)
+            print(f'# http://pets.fzu.cz/ ', file=f)
+            print(f'', file=f)
+            print(f'geometry continuous', file=f)
+            print(f'lambda {self.wavelength}', file=f)
+            print(f'Aperpixel {self.pixelsize}', file=f)
+            print(f'phi {float(self.osc_angle)/2}', file=f)
+            print(f'omega {omega}', file=f)
+            print(f'bin 1', file=f)
+            print(f'reflectionsize 15', file=f)
+            print(f'noiseparameters 25 10', file=f)
+            print(f'i/sigma    5.00   10.00', file=f)
+            print(f'', file=f)
+            # print("reconstructions", file=f)
+            # print("endreconstructions", file=f)
+            # print("", file=f)
+            # print("distortions", file=f)
+            # print("enddistortions", file=f)
+            # print("", file=f)
```

Comment:
Remove unused code.

```suggestion
            # print("reconstructions", file=f)
            # print("endreconstructions", file=f)
            # print("", file=f)
            # print("distortions", file=f)
            # print("enddistortions", file=f)
            # print("", file=f)
```

Reply:
same as above, thanks for the clean-up :)

Reply:
You are to make these changes yourself, I cannot commit to your branch. Github makes it easy to suggest changes through the UI. 

This means you can click on 'commit suggestion', and then it will be updated in your branch directly. Then you run 'git pull' on your computer, and it will pull the changes.

The other way is to make the edit yourself on your computer, and then push the changes back :-)

Does that help?

Reply:
thanks a lot! i was just confused since due to some reason the 'commit suggestion' button was greyed out and i couldn't click it?! anyway, as you suggested, I made the changes locally and pushed them, so it should be fine now? :)